tests/ui/search-page.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { test, expect } from "playwright/test";
+const callAPI = require("./api-call.spec.ts");
+
+test("search and check URL 2", async ({ page, baseURL }) => {
+  await page.goto(`/objekty`);
+  await page.locator(".ui.input input").fill("sklo");
+  await page.locator(".ui.input input").press("Enter");
+  await expect(page).toHaveURL(/q=sklo/);
+
+  await page.locator(".ui.input input").fill("sklo");
+  await page.locator(".ui > button:nth-child(2)").click();
+  await expect(page).toHaveURL(/q=sklo/);
+});
+
+test("redirection to create page", async ({ page, baseURL }) => {
+  await page.goto(`/objekty`);
+  await page.locator(".aside .secondary.button").click();
```

Comment:
Terrible selector, what if the button changes color? What if  more secondary buttons added to page